bin/run_tests.py

```diff
@@ -23,7 +23,12 @@
     print('Testing projects:', test_projects)
 
     run_test_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'run_test.py')
+    test_failed = 0
     for project_path in test_projects:
-        subprocess.check_call([sys.executable, run_test_path, project_path])
+        if 0 != subprocess.call([sys.executable, run_test_path, project_path]):
+            test_failed += 1
 
+    if test_failed > 0:
+        print('%d of %d projects built failed.' % (test_failed, len(test_projects)))
```

Comment:
1) What are these changes doing?
2) Why old %-formatting rather than format strings?

Reply:
1. write how many test group failed. 
2. Because bellow there is line `print('%d projects built successfully.' % len(test_projects))` and I would like to use same style.

Reply:
1. I meant in general. But I now read your comment that you don't want to stop on the first error.

2. Oh, I didn't see that. But then I vote for changing them all to fmt-style, if there aren't too many :-)